Algorithm/QCAlgorithm.cs

```diff
@@ -2888,13 +2888,25 @@ public long ShortableQuantity(Symbol symbol)
         /// The date is used to create a Symbol with the ticker set to the ticker the asset traded under on the trading date.
         /// </param>
         /// <returns>Symbol corresponding to the ISIN. If no Symbol with a matching ISIN was found, returns null.</returns>
-        [DocumentationAttribute(HandlingData)]
-        [DocumentationAttribute(SecuritiesAndPortfolio)]
+        [Documentation(HandlingData)]
+        [Documentation(SecuritiesAndPortfolio)]
```

Comment:
nit- let's just keep using `DocumentationAttribute` so it's consistent. If we need to we can rename them all at the same time in it's own PR